lib/mint/application.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Mint.Application do
+  @moduledoc false
+  use Application
+
+  def start(_type, _args) do
+    Application.put_env(:mint, :persistent_term, Code.ensure_loaded?(:persistent_term))
```

Comment:
You can accomplish @whatyouhide's [comment](https://github.com/elixir-mint/mint/pull/275/files#r488517000) by checking function was exported:
```suggestion
    Application.put_env(:mint, :persistent_term, function_exported?(:persistent_term, :get, 2))
```

Reply:
Yeah but then we'd cut out users on the OTP version that has get/1 from using the cache. We'd just do that to simplify our code so personally pretty strongly against it :)

Reply:
Agreed. I forgot that get/2 is just the same as get/1 but with default value, which was addressed by your [suggestion originally](https://github.com/elixir-mint/mint/pull/275#discussion_r488517000) btw 😅